spatialmath/mesh_test.go

```diff
@@ -77,52 +78,449 @@ func TestMeshTransform(t *testing.T) {
 }
 
 func TestMeshCollidesWithMesh(t *testing.T) {
-	mesh1 := makeSimpleTriangleMesh()
-
-	// Test collision with overlapping mesh
-	mesh2 := makeTestMesh(NewZeroOrientation(), r3.Vector{X: 0.5, Y: 0.5, Z: 0},
+	mesh1 := makeTestMesh(NewZeroOrientation(), r3.Vector{},
 		[]*Triangle{NewTriangle(
 			r3.Vector{X: 0, Y: 0, Z: 0},
 			r3.Vector{X: 1, Y: 0, Z: 0},
 			r3.Vector{X: 0, Y: 1, Z: 0},
 		)})
 
-	collides, err := mesh1.CollidesWith(mesh2, defaultCollisionBufferMM)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, collides, test.ShouldBeTrue)
+	// A mesh has 3 parts: {vertex, edge, face} ==> 6 possible basic collisions, accounting for symmetry
+
+	// v-v
```

Comment:
I assume `v-v` means `vertex-vertex`?
Can we please expand from shorthand to be more verbose?
Same for `v-e`, `v-f`, `e-e`, etc..